modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -221,7 +221,7 @@ def schemas(api_name: nil)
         hlr_create_schemas,
         hlr_response_schemas,
         generic_schemas.except(*%i[errorWithTitleAndDetail timeStamp X-Consumer-Username X-Consumer-ID documentUploadMetadata]),
-        shared_schemas
+        shared_schemas.slice(*%w[address phone timezone nonBlankString])
```

Comment:
This explicit `slice` update is so that the new `icn` shared schema isn't included